docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md

```diff
@@ -1,3 +1,37 @@
+# Test AgentD parametrized reconnection
+
+## Overview
+
+These tests will check that the connection is finally made when
+there are delays between connection attempts to the server.
+
+## Objective
+
+The objective is to check how AgentD behaves when there are delays between connection attempts
+to RemoteD using `TCP` and `UDP` protocols.
```

Comment:
```suggestion
The objective is to check how wazuh-agentd behaves when there are delays between connection attempts
to wazuh-remoted using `TCP` and `UDP` protocols.
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca